### DISCUSSION TITLE
Supabase Auth CORS error

### DISCUSSION THREAD

**Discussion by MrVibe** [supabase/supabase#26795] [Questions] [answered]
I am using Supabase Docker but the Auth is no longer working. Where do I configure this.
 this is the "Authorisation-settings" part that is missing in self hosted version.

/login:1 Access to fetch at 'https://studio.SUPABASE_URL/auth/v1/token?grant_type=password' from origin 'https://SITE' has been blocked by CORS policy: Request header field x-client-info is not allowed by Access-Control-Allow-Headers in preflight response.

**Comment by TheLexoPlexx** [comment]
This seems to be the same issue I encountered in #27814

**Comment by MrVibe** [comment, marked as answer]
I would like to update on this, I found out that my supabase url was incorrect. Somehow the self hosted Supabase version does not show your correct supabase URL in the supabase studio. So, I had to retrace my steps in the setup to locate the supabase URL.

**Reply by TheLexoPlexx** [reply]
Well, what is the supabase-URL supposed to  be? localhost? The ip of the server?

**Reply by TheLexoPlexx** [reply]
Found it. I removed the port, that confused CORS apparently.

**Reply by wild-deer** [reply]
hello ,i have the same problem ,can you share about how to fix this problem
